__tests__/app.test.js

```diff
@@ -121,3 +121,24 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("status:200, responds with an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toBeInstanceOf(Array);
+        users.forEach((user) => {
```

Comment:
When using a for each it is good practice to assert that the array has at least one item or more in it. Consider this matcher https://jestjs.io/docs/expect#toBeGreaterThan